### PR TITLE
OSDOCS-1203 Samples Operator Alerts

### PR DESCRIPTION
**Pull request by bmcelvee**
https://issues.redhat.com/browse/OSDOCS-1203

Related to https://github.com/openshift/openshift-docs/pull/26134.

### CODE REVIEW

**Comment by bmcelvee** [comment]
@wzheng1 would you mind reviewing this PR, please? Thank you! This PR is related to https://github.com/openshift/openshift-docs/pull/26134 and adds the 4.6-specific update about alerts.

**Comment by wzheng1** [comment]
@xiuwang  Could you please review? Thanks!

**Comment by xiuwang** [comment]
LGTM

File: modules/installation-restricted-network-samples.adoc [hunk 1]
```diff
@@ -111,9 +111,7 @@ Samples Operator configuration object.
 +
 [NOTE]
 ====
-Any unmirrored imagestreams that are not skipped, or if the Samples Operator is
-not changed to `Removed`, will result in the Samples Operator reporting a
-`Degraded` status two hours after the imagestream imports start failing.
+The Cluster Samples Operator issues alerts if imports are failing but the Cluster Samples Operator is periodically retrying them, or if imports are failing and the Cluster Samples Operator does not appear to be retrying them.
```
Comment (kalexand-rh) [inline modules/installation-restricted-network-samples.adoc:114]:
```suggestion
The Cluster Samples Operator issues alerts if imagestream imports are failing but the Cluster Samples Operator is either periodically retrying or does not appear to be retrying them.
```

Comment (bmcelvee) [reply modules/installation-restricted-network-samples.adoc:114]: Thanks!

**Pull request merged**